src/lib/y2partitioner/widgets/disk_table.rb

```diff
@@ -22,13 +22,15 @@ def initialize(disks, pager)
 
       # table items. See CWM::Table#items
       def items
+        probed_graph = Y2Storage::StorageManager.instance.y2storage_probed
```

Comment:
so basically for this DiskTable constructor have to have these 4 parameters: 

1. pager for double click feature. Fine from UI POV

2. disks to display. It make sense for me, we want show some object, so we pass it.

3. old graph to recognize if there is change in object. My OOP feeling is that it is a bit wrong and object itself should know it.

4. new graph to allow creating copy of it ( we already discussed it with @mvidner and @teclator that we probably make it global singleton to avoid passing it everywhere ). Again my OOP feeling is a bit bad here, as I would like to have possibility to do disk.dup, modify it and when needed, somehow make it effective change or scratch it.